### ISSUE TITLE
Dialog content hidden from screen readers

### ISSUE THREAD

**Issue by Malvoz** [Canadian-Geospatial-Platform/GeoView#189]
The heading "Keyboard Navigation" and description "We'll keep focus locked on the viewer with SHIFT + ESCAPE" that is the content of a dialog after tabbing into the map is not accessible using a screen reader (tested using [NVDA](https://www.nvaccess.org/)) due to `aria-hidden="true"` on an ancestor element (`div` with class `jss1`):

![aria-hidden-dialog](https://user-images.githubusercontent.com/26493779/114187514-60766900-9948-11eb-8af0-06bfadd3ec33.png)

**Comment by jolevesq** [comment]
I tested with VoiceOver on my Mac and I am now able to access the text content of my dialog box. @Malvoz Is this link https://jolevesq.github.io/GeoView/basemaps.html when you Tab to map solved the problem?

Thanks again for your helpful comments

**Comment by Malvoz** [comment]
Yes the content was announced properly, tested using NVDA in Chrome. Nice!

Side note: I initially pressed <kbd>Enter</kbd> to enable keyboard navigation, but that didn't work, only <kbd>Space</kbd> did. 
https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-3 (recommended keyboard interactions for buttons) suggests that both should work:

> When the button has focus:
>
> <kbd>Space</kbd>: Activates the button.
> <kbd>Enter</kbd>: Activates the button.
